samples/sampler/stories/default/Panel.js

```diff
@@ -19,7 +20,7 @@ storiesOf('Agate', module)
 					subtitle={text('subtitle', HeaderConfig, 'Header Subtitle')}
 					title={text('title', HeaderConfig, 'Header Title')}
 				/>
-				<Button>Click me</Button>
+				<Button onClick={action('onClick')}>Click me</Button>
```

Comment:
Let's remove the action `onClick`. We actually add the `action` of its components interest. This means we need to add only `forward` event of the component which app developer need to know.
Button click is not the Panel's